src/main/scala/strawman/collection/ArrayOps.scala

```diff
@@ -27,7 +27,7 @@ class ArrayOps[A](val xs: Array[A])
   protected[this] def fromTaggedIterable[B: ClassTag](coll: Iterable[B]): Array[B] = coll.toArray[B]
   protected[this] def fromSpecificIterable(coll: Iterable[A]): Array[A] = coll.toArray[A](elemTag)
 
-  protected[this] def newSpecificBuilder() = new GrowableBuilder(ArrayBuffer.empty[A]).mapResult(_.toArray(elemTag))
+  protected[this] def newSpecificBuilder() = ArrayBuffer.newBuilder[A]().mapResult(_.toArray(elemTag))
```

Comment:
This could avoid copying if the size matches. We may also consider passing a size hint to `newSpecificBuilder` (but it's probably not as useful in the new design where most operations use `fromSpecificIterable` instead).